variablecompany/bite
# feat: add the dish detail screen for each dish in the menu

## Describe your changes

This adds the dish detail screen for each dish, whenever coming from the menu or from pressing the home screen. If you press on the detail you can view the ingredients used for that particular dish and allergens. 

At the bottom you have the incremental button and the add button which will add the item to the store of the cart

## How to test

- Go to Home or go to the menu screen
- Press on each dish items
- Verify that it takes you to the screen
- Press on incremental button
- Press on add cart
- Verify that the cart receives the item

## Checklist before requesting a review
